installer.sh

```diff
@@ -15,37 +15,80 @@ MSWEBVIEW_VERSION=1.0.3351.48
 MSWEBVIEW_INCLUDE_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/include
 MSWEBVIEW_LIB_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/x64
 
+
+
+CHROMIUM_VERSION=138.0.17
+CHROMIUM_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/include
+CHROMIUM_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/x64
+
 SYS_OPTS=""
 
 PATH=$(cygpath -w /ucrt64/bin):$PATH
 export PATH
-
 source ./.scripts/util.sh
 # For Windows, check that we are running in a MSYS2 UCRT environment
 check_env
 
 PACKAGE_INSTALL_HELP="\
 #  Installs required library packages.
-#  ATM only MSWEBVIEW2 on Windows is supported, so we leave this as basic logic.
-#  @todo Expand the logic for mutiple OS and embedded browser packages.
+#  ATM only MSWEBVIEW2 on Windows is fully supported, so we leave this as basic logic.
+#  The installer currently supports CEF 
+#  @todo Expand the logic for multiple OS and embedded browser packages.
 ## Usage:
 #   packages_install\
 "
+
+select_browser_caller() {
+    if [ "$(is_help "$@")" = true ]; then
+        print_help "$SELECT_BROWSER_HELP"
+        [ $PS1 ] && return 0 || exit 0
+    fi
+    if [[ -z "$1" ]]; then
+        throw_error "Browser type must be specified. Use: Chromium or MSWebView2"
+        return 1
+    fi
+    source ./.scripts/util.sh
+    select_browser "$1"
+
+}
+
+PACKAGE_INSTALL_HELP="\
+#  Installs required library packages.
+#  Supports both CEF (Chromium) and MS WebView2 browser engines.
+#  Browser must be selected first using select_browser command.
+## Usage:
+#   packages_install\
+"
```

Comment:
Move `SELECT_BROWSER_HELP` directly above it's function so that the coding pattern remains consistent.

Ensure that capitilisation is consistent for the options: "Chromium" or "chromium"? The user will be confused by the ambiguity. I suggest lower case consistently for this context.